Project-1/README.md

```diff
@@ -13,63 +13,300 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
+	
   (e)  T that says that x has exactly two brothers 
+  	hasTwoBrothers ≡ Person⊓≤2brother_of.Person.⊓≥2brother_of.Person. This would work if I had a brother_of relation... but alas. 
+  
   ```
 
 
 5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
 
+	Since I don't really have any acceptable formulas... 
+
+
 6. Explain the difference - using natural language - between the first-order prefixes:
   ```
   (a) ∃x∀y and ∀x∃y
+	‘∃x∀y’ is equivalent to ‘There is an x for all y…’ while 
+        ‘∀x∃y’ is equivalent to ‘For all x, there is a y…’ 
+
+           That is, The former means that there is at least one y to which all xs have some sort of 
+	relation (to be specified after the prefix), while the latter means that every x has a(n unspecified) 
+	relation to some y, which does not have to be the same y for every x (though it certainly could be, 
+	depending upon how the relation is specified).
+	   
   (b) ∃x∀y∃z and ∀x∃y∀z 
+  	'∃x∀y∃z' is equivalent to 'There exists an x for all y such that there exists a z such that...'
+	'∀x∃y∀z' is equivalent to 'For all x there exists a y for all z such that...'
+		
+	   That is, the former means that there is at least one x to which all things y bear some sort of 
+	relation, which is in a relation to another existing thing, z and the latter means that all xs 
+	have some relation (yet to be specified) to some class (min 1) of existing thing(s), related also to all things z. 
+ 
+  
   (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+  	'∀x∃y∀z∃w' is equivalent to 'For all x there exists a y such that for all z there exists a w such that...'
+	'∃x∀y∃z∀w' is equivalent to 'There exists an x for all y such that there exists a z for all w such that...'
+		
+	   That is, the former means that all xs bear some relation to at least one y which bear(s) a relation to all 
+	zs which bear a relation to some w.The latter means that some thing (x) is related to all y, which each have some 
+	unspecified relation to some other z, which is then related to all w. 
+  
 ```
 	
 7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
 ```
 ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
 ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+
+	U: {0, 1, 2}
+	R: {(0,2), (1,0), (1,1), (1,2), (2,2)}
+	
+
+In this model, the first sentence is false while the second sentence is true. 
+
 ```
 	
 8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
   ```
   (a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
-  (b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+  
+			1.¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+			2.¬∀y(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))(1)
```

Comment:
What rule justifies moving from ¬∀x∀y to ¬∀y? 

Note, I'm not asking about the irrelevant ∀y, but rather about instantiating ∀ within the scope of ¬.